src/worker.jl

```diff
@@ -172,12 +172,9 @@ function process_message(msg::Solve,
 
     _execute_subproblem_callbacks(sub, msg.niter, msg.scen)
 
-    if record.warm_start
-        warm_start(sub.problem)
-    end
-
     start = time()
     sts = solve_subproblem(sub.problem)
+    record.warm_start && warm_start(sub.problem)
```

Comment:
A couple of things here. First warm starting shouldn't be done within the solution timing bracket.  That is you should swap the warm start call and stop time assignment.

Second, this is probably a result of all my experience with confusing C/C++ code but I don't like using `&&` in place of `if` statements.  I find it makes the code harder to follow.